src/diffusers/schedulers/scheduling_utils.py

```diff
@@ -45,7 +45,7 @@ class SchedulerMixin:
     def set_format(self, tensor_format="pt"):
         deprecate(
             "set_format",
-            "0.6.0",
+            "0.7.0",
             "If you're running your code in PyTorch, you can safely remove this function as the schedulers are always"
             " in Pytorch",
         )
```

Comment:
We can either bump or deprecate this function, but I'm ok with keeping it a bit longer